src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,86 @@
+package step2;
+
+public class Calculator {
+    public int addition (int num1, int num2) {
+        return Operator.PLUS.caculate(num1, num2);
+    }
+
+    public int subtraction (int num1, int num2) {
+        return Operator.MINUS.caculate(num1, num2);
+    }
+
+    public int multiplication (int num1, int num2) {
+        return Operator.MUL.caculate(num1, num2);
+    }
+
+    public int division (int num1, int num2) throws ArithmeticException {
+        if (num2 <= 0) {
+            throw new ArithmeticException("0으로는 나눌수 없습니다");
+        }
+        return Operator.DIV.caculate(num1, num2);
+    }
+
+    public int stringCalculator (String op,int num1,int  num2) {
+        if ("+".equals(op)) {
+            return Operator.PLUS.caculate(num1, num2);
+        }
+        if ("-".equals(op)) {
+            return Operator.MINUS.caculate(num1, num2);
+        }
+        if ("*".equals(op)) {
+            return Operator.MUL.caculate(num1, num2);
+        }
+        if ("/".equals(op)) {
+            return Operator.DIV.caculate(num1, num2);
+        }
+        throw new IllegalArgumentException(ErrorMsg.ERROR_CODE_2);
+    }
+    public void vaildate (String input) {
+        if (input.trim().isEmpty() || input == null) {
+            throw new IllegalArgumentException(ErrorMsg.ERROR_CODE_1);
+        }
+    }
+
+    public void operatorCheck (String input) {
+        if (!"+".equals(input) && !"-".equals(input) && !"*".equals(input) && !"/".equals(input)) {
+            throw new IllegalArgumentException(ErrorMsg.ERROR_CODE_2);
+        }
+    }
+
+    public String[] splitCheck (String input) {
+        return input.split(" ");
+    }
+
+    public int changeResult (String input) {
+        String[] st = input.split(" ");
+        int result = 0;
+
+        for (int i = 0; i < st.length; i += 2) {
+            result = Integer.parseInt(st[i]);
+        }
+        System.out.println(result);
+        return result;
+    }
+
+    public String changeString (String input) {
+        String[] st = input.split(" ");
+        String operator = "";
+        for (int i = 1; i < st.length - 1; i += 2) {
+            operator = st[i];
+            System.out.println(operator);
+        }
+        return operator;
+    }
+
+    public int changeAll (String input) {
+        String[] st = input.split(" ");
+        int result = Integer.parseInt(st[0]);
+
+        for (int i = 1; i < st.length - 1; i += 2) {
+            String operator = st[i];
+            stringCalculator(operator,result, Integer.parseInt(st[i + 1]));
```

Comment:
```suggestion
            String operatorString = st[i];
            Operator operator = Operator.getOperator(operatorString);
            result = operator.caculate(result, Integer.parseInt(st[i + 1]));
```
위와 같이 바꿔주면 연산은 입력된 연산 String에 맞는 Operator enum에게 위임할 수 있게 됩니다.
if else 없이 연산 String에 맞는 Operator에서 각자 알맞은 연산을 해서 결과를 반환하게 되죠.
이렇게 하면
addition
subtraction
multiplication
division
stringCalculator
operatorCheck
와 같은 이 클래스 위쪽에 있는 메서드들이 전부 필요 없어지게 되죠 😊
이렇게 반영해주셨으면 했답니다 😄

문자열 계산기라는 미션 특성상 객체 분리가 어색할 수도 있습니다. 본격적인 자동차 미션부터는 더 직관적으로 객체를 분리하실 수 있으니 함께 고민해보시죠! :)